test/linter/test-versions.js

```diff
@@ -161,8 +175,12 @@ function testVersions(filename) {
   const logger = new Logger('Versions');
 
   /**
-   * @param {Identifier} data
-   * @param {string} [relPath]
+   * Process the data for version errors
+   *
+   * @param {Identifier} data The data to test
+   * @param {Logger} logger The logger to ouptut errors to
+   * @param {string} [relPath] The path of the data
```

Comment:
Why is `relPath` in brackets here (but not in `testVersions`'s JSDoc)?

```suggestion
   * @param {string} relPath The path of the data
```